src/index.js

```diff
@@ -200,19 +197,36 @@ function shimHasMany(target) {
 
   shimmer.wrap(target, 'get', original => {
     return function bathedGetHasMany(instances, options = {}) {
+      let isCount = false;
       if (options.include || options.transaction) {
         return original.apply(this, arguments);
       }
 
+      const attributes = options.attributes;
```

Comment:
This is not very pretty, but I think it'll have to do

The other alternative is to shim the  `count` function - Two problems however: We would have to duplicate the code, and it is only exposed on `hasMany`, so would require a sequelize PR ;)